terraform_files/vsphere/main.tf

```diff
@@ -0,0 +1,93 @@
+provider "vsphere" {
+  user                 = var.vsphere_username
```

Comment:
Elaborate on the description of each data section.
In addition, I'd be glad if you could add a README providing an overview about using `vsphere` provider.

Reply:
It fully documented on the terraform provider why to document it here? I have to document the variables i have created or special behavior